README.md

```diff
@@ -37,45 +36,48 @@ Foundry Starter Kit is a repo that shows developers how to quickly build, test,
 
 Please install the following:
 
--   [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)  
-    -   You'll know you've done it right if you can run `git --version`
--   [Foundry / Foundryup](https://github.com/gakonst/foundry)
-    -   This will install `forge`, `cast`, and `anvil`
-    -   You can test you've installed them right by running `forge --version` and get an output like: `forge 0.2.0 (f016135 2022-07-04T00:15:02.930499Z)`
-    -   To get the latest of each, just run `foundryup`
-
-And you probably already have `make` installed... but if not [try looking here.](https://askubuntu.com/questions/161104/how-do-i-install-make)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+  - You'll know you've done it right if you can run `git --version`
+- [Foundry / Foundryup](https://github.com/gakonst/foundry)
+  - This will install `forge`, `cast`, and `anvil`
+  - You can test you've installed them right by running `forge --version` and get an output like: `forge 0.2.0 (f016135 2022-07-04T00:15:02.930499Z)`
+  - To get the latest of each, just run `foundryup`
 
 ## Quickstart
 
 ```sh
 git clone https://github.com/smartcontractkit/foundry-starter-kit
 cd foundry-starter-kit
-make # This installs the project's dependencies.
-make test
 ```
 
-## Testing
+## Install dependencies as follows:
 
-```
-make test
-```
+You can add the `--no-commit` flag to each of these `install` commands if your workspace has uncommitted changes.
```

Comment:
If I follow these instructions from a clean repo, only having checked out this branch, I receive the following error message when running this command (and any of the `forge install` commands):
```
$ forge install foundry-rs/forge-std
Installing forge-std in /Users/thomas/workspace/foundry-starter-kit/lib/forge-std (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
fatal: 'lib/forge-std' already exists and is not a valid git repo
Error: 
git submodule exited with code 128
```

If I remove the `lib/forge-std` directory, I can then run the command with the `--no-commit` flag, but this seems like the wrong approach.

The better fix I found would be simply to change the instructions to say to run `forge install`, then all these components are installed without issue.

Reply:
fixed